README.md

```diff
@@ -31,12 +32,17 @@
 ### Runners
 
 - [jest-runner-eslint](https://www.github.com/jest-community/jest-runner-eslint) ESLint runner for Jest.
+- [jest-runner-mocha](https://github.com/rogeliog/jest-runner-mocha) Mocha runner for Jest.
 
 ### Snapshot
 
 - [snapshot-diff](https://www.github.com/jest-community/snapshot-diff) Takes two values, and return their difference as a string, ready to be snapshotted with toMatchSnapshot(). Especially helpful when testing the difference between different React component states.
 - [jest-snapshots-svg](https://www.github.com/jest-community/jest-snapshots-svg) Take a React component tree, and render it into an SVG.
 
+### Migration
+
+- [jest-codemods](https://github.com/skovhus/jest-codemods)
```

Comment:
@skovhus ideas for text? 🙂

Reply:
"Makes it easy to migrate to and from Jest from other test runners"

Reply:
I'll take it 🙂

Reply:
As we don’t migrate from Jest (yet) I guess something like

“Makes it easy migrate from other test runner and frameworks to Jest.”

And thanks for updating this!

Reply:
Ah, that's a very good point 😅